configs/scripts/program/dump.sh

```diff
@@ -16,7 +16,7 @@ cd $(dirname $(dirname $(dirname $SCRIPT_DIR)))
 OUTPUT=$1
 
 if [ -z ${RPC+x} ]; then
-    RPC="https://api.mainnet-beta.solana.com"
+    RPC="https://api.devnet.solana.com"
```

Comment:
hmm I think in general we want to be testing with mainnet-beta programs.  But I think spl-account-compression you must use the devnet version for this.  We did this in mpl-core with the oracle-test-program on devnet, I think we just cloned the script.  Can you do that here as well?